index.html

```diff
@@ -30,7 +31,14 @@
     </style>
 </head>
 <body>
-    <div class="container">
+    <header class="header">
+			
+		<nav class="teal lighten-2">
```

Comment:
we will be using the root classes that are set in the style.css sheet... For an example please look here: https://github.com/peterkim1025/distract_me/blob/77cf85029d7d38db31c2852a534112ab6c47c0fb/assets/html/catPage.html#L17